### ISSUE TITLE
[Feature Request] Add support for MODX Git installations

### ISSUE THREAD

**Issue by pixelchutes** [modmore/Gitify#105]
For those choosing to operate their MODX instances via [Git installation] (http://rtfm.modx.com/revolution/2.x/getting-started/installation/git-installation), it would be valuable if Gitify could support that as part of the install workflow. Perhaps a way to target a given branch and/or sha hash? Thoughts?

**Comment by Mark-H** [comment]
Yeah that would definitely be valuable. Pull requests are welcome too :D
